pkgs/applications/emulators/craftos-pc/wrapper.nix

```diff
@@ -0,0 +1,63 @@
+# Shamelessly stolen and modified from mpv/wrapper.nix
+# Arguments that this derivation gets when it is created with `callPackage`
+{ stdenv
+, lib
+, makeWrapper
+, symlinkJoin
+}:
+
+# the unwrapped CraftOS-PC derivation - 1st argument to `wrapCraftos`
+craftos-pc:
+
+let
+  # arguments to the function (exposed as `wrapMpv` in all-packages.nix)
+  wrapper = {
+    extraMakeWrapperArgs ? [],
+    # a set of derivations (probably from `craftosPlugins`) where each is
+    # expected to have a `pluginName` passthru attribute that points to the
+    # name of the script that would reside in the script's derivation's
+    # `$out/share/craftos/plugins/`.
+    plugins ? [],
+  }:
+  let
+    # All arguments besides the input and output binaries (${craftos-pc}/bin/craftos and
+    # $out/bin/craftos). These are used by the darwin specific makeWrapper call
+    # used to wrap $out/Applications/CraftOS-PC.app/Contents/MacOS/craftos as well.
+    mostMakeWrapperArgs = lib.strings.escapeShellArgs ([ "--inherit-argv0"
+    ] ++ (lib.lists.flatten (map
+      # For every script in the `scripts` argument, add the necessary flags to the wrapper
```

Comment:
Do we have an alternative to that? We could easily trigger the arg length limit here?

Reply:
I can't find an alternative.